src/openms/source/ANALYSIS/OPENSWATH/OpenSwathScoring.cpp

```diff
@@ -86,8 +104,28 @@ namespace OpenMS
     std::vector<double> normalized_library_intensity;
     getNormalized_library_intensities_(transitions, normalized_library_intensity);
 
+    // automatically compute the amount of spectra to add based on the fraction of the retention time peak width, or add a fixed number of spectra
+    int n_merge_spectra = 1;
+    if (spectra_merge_method_type_ == SpectrumMergeMethodType::DYNAMIC)
+    {
+        double leftWidth = imrmfeature->getMetaValue("leftWidth");
+        double rightWidth = imrmfeature->getMetaValue("rightWidth");
+        double peakWidth = rightWidth - leftWidth;
+        n_merge_spectra = std::max(1, static_cast<int>(std::ceil(peakWidth * merge_spectra_by_peak_width_fraction_)) | 1);
```

Comment:
Same here. You would get rid of the hacky | 1

Reply:
ah ok I see that you now have separate arguments in the initialize method.
I think it might make sense to have a quick zoom call at some point so I can better explain how this could be approached. I am terrible at explaining this as part of a GitHub issue :)

Reply:
Sounds good. I just made another PR for IM peak picking, I have to make one more PR for the rest of the IM-IPF improvements. Maybe we can have a zoom call after I've made the 3rd PR, so if there are other suggestions/fixes for those, then we can discuss those at the same time as well? Maybe this Wednesday or Thursday?

Reply:
Works for me. I am Meeting with Josh on Thursday anyway so maybe we Can scheffle around that time?

Reply:
> Works for me. I am Meeting with Josh on Thursday anyway so maybe we Can scheffle around that time?

Sounds good. @jcharkow  could we use the same zoom room?

Reply:
Yes we can use the same zoom room. Room+Time is in the lab calendar.